report-ng/app/src/components/timings/session-timings/session-timings.ts

```diff
@@ -0,0 +1,228 @@
+    private _testDuration;
+    private _bars;
+
+
+    constructor(
+        private _statusConverter: StatusConverter,
+        private _statisticsGenerator: StatisticsGenerator,
+    ) {
+        super();
+        this._sessionMetrics = [];
+        this._executionStatistics = [];
+        this._methodDetails = [];
+        this._bars = [];
+    }
+
+    activate(params: any, routeConfig: RouteConfig, navInstruction: NavigationInstruction) {
+        super.activate(params, routeConfig, navInstruction);
+
+        this._statisticsGenerator.getExecutionStatistics().then(executionStatistics => {
+            this._executionStatistics = executionStatistics;
+
+            this._testDuration = [this._executionStatistics.executionAggregate.executionContext.contextValues.startTime,
+                this._executionStatistics.executionAggregate.executionContext.contextValues.endTime]
+
+            executionStatistics.classStatistics
+                .forEach(classStatistic => {
+                    let methodContexts = classStatistic.methodContexts;
+                    methodContexts = methodContexts.filter(methodContext => methodContext.methodType == MethodType.TEST_METHOD);
+
+                    let methodDetails = methodContexts.map(methodContext => {
+                        return new MethodDetails(methodContext, classStatistic);
+                    });
+                    methodDetails.forEach(methodDetails => {
+                        this._methodDetails.push(methodDetails);
+                    })
+                })
+        });
+
+        const sessionInformationArray = [];
+
+        this._statisticsGenerator.getSessionMetrics().then(sessionMetrics => {
+            this._sessionMetrics = sessionMetrics;
+            sessionMetrics.forEach(metric => {
+                const methodList = [];
+                this._methodDetails.forEach(method => {
+                    if (method.methodContext.sessionContextIds.includes(metric.sessionContext.contextValues.id)) {
+                        methodList.push(method);
+                    }
+                });
+
+                const sessionInformation: ISessionInformation = {
+                    sessionName: metric.sessionContext.contextValues.name,
+                    sessionId: metric.sessionContext.sessionId,
+                    browserName: metric.sessionContext.browserName,
+                    browserVersion: metric.sessionContext.browserVersion,
+                    server: metric.sessionContext.serverUrl,
+                    node: metric.sessionContext.nodeUrl,
+                    methodList: methodList,
+                    sessionDuration: this._calculateDuration(metric.metricsValues[0].startTimestamp, metric.metricsValues[0].endTimestamp),
+                    baseurlDuration: this._calculateDuration(metric.metricsValues[1].startTimestamp, metric.metricsValues[1].endTimestamp),
+                    startTime: metric.metricsValues[0].startTimestamp,
```

Comment:
Why is `startTime` the only of these values that isn't displayed in the tooltip?